src/store/holochain.js

```diff
@@ -58,13 +58,6 @@ const initializeClientHolo = async (commit, dispatch, state) => {
     return
   }
 
-  const appInfo = await holoClient.appInfo()
-  const [cell] = appInfo.cell_data
-  const { cell_id: cellId, cell_nick: dnaAlias } = cell
-  commit('setHoloClientAndDnaAlias', { holoClient, dnaAlias })
-  const [dnaHash] = cellId
-  commit('setDnaHash', 'u' + Buffer.from(dnaHash).toString('base64'))
```

Comment:
Why are you moving all this stuff to after sign in? Now I think there's gonna be a lot of errors in the console when it tries to call list_messages and holoClient isn't set

Reply:
When this logic is prior, the ec ui would actually display prior to signing-in/signing-up, which made for a bit of a broken experience imo, but also could be a confusing for a web user about why they could see the screen prior (as an anonymous agent) prior to the web user signing in.  When I tested it out, `list_messages` only triggered once prior to this... which I thought was minimal cost to avoid jumping screens between the anonymous and signed-in user.   

...But, I'd like to ask why we would like to call `list_messages` prior to signing in anyway?  Shouldn't we be making that call once we are connected to the correct instance?

Reply:
We had a PL on this and agreed as a team that a landing page would be ideal for later on and that for now we would just update the placement of the list_channels call (in a different PR).